src/components/Playground.tsx

```diff
@@ -97,9 +101,7 @@ class Playground extends React.Component<IPlaygroundProps, PlaygroundState> {
   }
 
   private chooseEditorValue(props: IPlaygroundProps): string {
-    return parsePrgrm(this.props) || this.props.editorValue === undefined
-      ? defaultPlaygroundText
-      : this.props.editorValue
+    return parsePrgrm(this.props) || this.props.editorValue
```

Comment:
As of 20f72c82d07933717c9f7c39455f330a7c50817d, the playground cannot be edited after the playground link is accessed. If i recall, it is because of how an empty string acts as `false` in the `||` check. Can you try to revert this line to see if it fixes it?